go.mod

```diff
@@ -18,16 +18,16 @@ require (
 	github.com/IBM/event-notifications-go-admin-sdk v0.6.1
 	github.com/IBM/eventstreams-go-sdk v1.4.0
 	github.com/IBM/go-sdk-core/v3 v3.2.4
-	github.com/IBM/go-sdk-core/v5 v5.17.3
+	github.com/IBM/go-sdk-core/v5 v5.17.4
 	github.com/IBM/ibm-cos-sdk-go v1.10.3
 	github.com/IBM/ibm-cos-sdk-go-config/v2 v2.1.0
 	github.com/IBM/ibm-hpcs-tke-sdk v0.0.0-20211109141421-a4b61b05f7d1
 	github.com/IBM/ibm-hpcs-uko-sdk v0.0.20-beta
 	github.com/IBM/keyprotect-go-client v0.14.0
 	github.com/IBM/logs-go-sdk v0.3.0
 	github.com/IBM/networking-go-sdk v0.47.1
-	github.com/IBM/platform-services-go-sdk v0.62.11
-	github.com/IBM/project-go-sdk v0.3.5
+	github.com/IBM/platform-services-go-sdk v0.64.3
+	github.com/IBM/project-go-sdk v0.3.4
```

Comment:
rolling back to previous version shd n't override other services go sdk's verison